devito/types/dense.py

```diff
@@ -588,11 +588,11 @@ def symbolic_shape(self):
     _C_typename = 'struct %s *' % _C_structname
     _C_field_data = 'data'
     _C_field_size = 'size'
-    _C_field_nopad_size = 'npsize'
-    _C_field_domain_size = 'dsize'
-    _C_field_halo_size = 'hsize'
-    _C_field_halo_ofs = 'hofs'
-    _C_field_owned_ofs = 'oofs'
+    _C_field_nopad_size = 'size_nopad'
```

Comment:
Its up to you but I strongly disagree with 'size_xxx'
It is also weird to have 
 _C_field_xxx_size = 'size_xxx'
Doesn't make sense IMHO